.github/workflows/test.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       - name: Maven Test
         id: test
         run: |
-          mvn test --file ./pom.xml
+          mvn test --file ./pom.xml -Dtest=!%regex[.*.integration.*] -DfailIfNoTests=false
```

Comment:
Why do we want to set `failIfNoTests` to false?

Reply:
Adding this caused to ignore the default **surefire:test** configuration, as result of that some tests where skipped
Anyway, I changed it to exclude the regex from the pom.xml configuration
Moved this change to different PR, please see PR #290